src/releasers/java-yoshi.ts

```diff
@@ -299,6 +299,18 @@ export class JavaYoshi extends ReleasePR {
         })
       );
     });
+
+    this.extraFiles.forEach(path => {
```

Comment:
my slight concern here is behavior that's one-language specific, perhaps add a TODO to flesh out this behavior. I'm imagining you'd eventually pass a file plus the updater to apply?

Reply:
I think that will be too much config. The general use case for extra files would be one-offs that update the version strings in a file. Each language will want a generic way to do this - we don't want a bunch of one-off updaters for this.